### DISCUSSION TITLE
The Slack channel is open now

### DISCUSSION THREAD

**Discussion by unix** [zeit-ui/react#572] [Show and tell]
We have opened up the [Slack channel](https://join.slack.com/t/geist-ui/shared_invite/zt-152hkla76-3j3bCfWRx7QlFHtIA3RXqQ) as an instant communication tool, which may be helpful for some specific feedback. (We will not reply to any of your technical questions in Slack, if so, please leave a question on Github.)

🚫 For the following types, please continue on GitHub so that other developers can search and refer to the context:

- [Feedback an issue](https://github.com/geist-org/react/issues/new?assignees=&labels=&template=1-bug-report.md)
- [Request a feature](https://github.com/geist-org/react/issues/new?assignees=&labels=&template=2-feature-request.md)
- [Need help or ask a question](https://github.com/geist-org/react/discussions/new)
- Others that should be recorded by search engines

👏 For the following types, you can join the slack channel to continue:

- Talk casually with others about Geist Org.
- Share with others the cool things you do with Geist UI.
- You want to give feedback on debugging content that is not suitably available for public display.

**Comment by agoldis** [comment]
This link is no longer active

**Reply by unix** [reply]
updated

**Comment by Arithmetics** [comment]
new link?

**Reply by unix** [reply]
updated

**Comment by brionmario** [comment]
Hi @unix,

The links seem to be broken again. Could you please update 😃 ?

**Comment by nordengg** [comment]
Hi @unix, the invite has been withdrawn.

**Comment by shellwhale** [comment]
Dead link
